### ISSUE TITLE
"Error: Failed to find python version from target process" on macOS

### ISSUE THREAD

**Issue by azag0** [benfred/py-spy#15]
I'm getting this error for a Pyenv-installed Python running an Ipython kernel. Running with `RUST_LOG=info`, I get
```
 INFO 2018-09-07T06:14:57Z: py_spy::python_spy: Getting version from python binary BSS
 INFO 2018-09-07T06:14:57Z: py_spy::python_spy: Failed to get version from BSS section: failed to find version string
 INFO 2018-09-07T06:14:57Z: py_spy::python_spy: Trying to get version from path: /Users/hermann/.pyenv/versions/3.6.6/Python.framework/Versions/3.6/Resources/Python.app/Contents/MacOS/Python
Error: Failed to find python version from target process
```

**Comment by benfred** [comment]
thanks for the bug report! I'm having some issues replicating though, I installed python 3.6.6 through pyenv on osx using ```pyenv install 3.6.6``` and everything worked.

I think your pyenv installation is different, it looks like you have a 'Python.app' style python distribution (like the system python/homebrew) instead of the one I got from pyenv which I believe just compiled the source. Do you remember how you installed it? My guess is that I'm failing to find the correct libpython from your distribution.

Also could you paste the full log? The libpython filename should be something like '/usr/local/Cellar/python/3.6.4_2/Frameworks/Python.framework/Versions/3.6/Python' (homebrew) or '/System/Library/Frameworks/Python.framework/Versions/2.7/Python' (system python).

**Comment by azag0** [comment]
I installed the Pyenv Python as a framework to have a Python shared library
```
env PYTHON_CONFIGURE_OPTS="--enable-framework" pyenv install 3.6.6
```

The full log from trying to run PySpy is here: https://gist.github.com/776cd807a34eb8d37b37a8547c282fe0

The Python app is, I think, in `/Users/hermann/.pyenv/versions/3.6.6/Python.framework/Versions/Current/Python`

**Comment by benfred** [comment]
thanks! I hadn't tested python built on OSX with "--enable-framework". I have a fix here: 08b98715caab060aeb5d664775511a513e2d96d8, and will be in the next release

**Comment by benfred** [comment]
fix is in v0.1.5

**Comment by azag0** [comment]
Thanks! Works for me now as well.
